gh/index.js

```diff
@@ -23,8 +23,13 @@ module.exports = yeoman.generators.Base.extend({
     var defaultName = path.basename(process.cwd());
     var prompts = [
       {
+        name: 'ghHostname',
+        message: 'What is your GitHub hostname?',
```

Comment:
Hm... testing this out. This line feels weird. For the majority of folks who aren't GitHub Enterprise users, or don't know about it, this is just a strange question.

I think instead of a prompt I'd prefer to pass this in as a flag.

`yo polymer:gh --hostname enterprise-foo.com`

@addyosmani @arthurvr wdyt?